cmd/sidecar/main.go

```diff
@@ -20,6 +22,12 @@ var infoLogger = log.InfoLogger
 
 func main() {
 	var err error
+	var secretsHandler *secrets.Secrets
+
+	const(
```

Comment:
is there a way to open this const up to other packages? so we can use a single const across our project?

Reply:
*interesting find*, it is not enough to just import, all variables that we want to export from another file need to be in *uppercase*
Ex:
in package `*storageConfig* ...storage/config` we have consts,
`
const (
	K8s  = "k8s_secrets"
	None = "none"
)
`
In order to use each consts outside this pkg , we need to capitalize them

Then in `sidecar/main.go` we import `*storageConfig* ...storage/config` and write storageConfig.K8s.

v cool stuff